keyboards/dmqdesign/spin/readme.md

```diff
@@ -0,0 +1,25 @@
+# SPIN
+
+![SPIN](https://i.imgur.com/5oYg1WD.jpg)
+
+The SPIN Macro Pad is a USB-C, hotswappable Macro Pad with 3 encoders designed by Nick from DMQ Design.
+
+Keyboard Maintainer: [Nicholas Junker](https://github.com/Quarren42)  
+
+Hardware Supported: SPIN Macro Pad PCB/kit
+
+Hardware Availability: The macro pad is available at [DMQdesign.com](www.dmqdesign.com)
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make DMQdesign/spin:default
+
+    OR
+    
+    make DMQdesign/spin:default:dfu
```

Comment:
```suggestion
    make dmqdesign/spin:default

    OR
    
    make dmqdesign/spin:default:dfu
```

Sorry, just noticed this.  Build instructions are case-sensitive so someone following the readme to the letter could get tripped up.

Reply:
Re-requested a review, thanks again for combing through everything.